gcn/notices/svom/alert.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/svom/alert.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "unevaluatedProperties": false,
+  "title": "SVOM Alert",
+  "description": "ECLAIRs or GRM trigger parameters computed onboard",
+  "allOf": [
+    { "$ref": "../core/AdditionalInfo.schema.json" },
+    { "$ref": "../core/Alert.schema.json" },
+    { "$ref": "../core/DateTime.schema.json" },
+    { "$ref": "../core/Event.schema.json" },
+    { "$ref": "../core/Localization.schema.json" },
+    { "$ref": "../core/Reporter.schema.json" },
+    { "$ref": "../core/Statistics.schema.json" }
+  ],
+  "properties": {
+    "$schema": true,
+    "notice_level": {
+      "enum": ["N1e", "N1g"],
+      "description": "SVOM notice reference id."
+    },
+    "slew_status": {
+      "enum": ["accepted", "rejected", "not_requested"],
+      "description": "Spacecraft slew status"
+    },
+    "trigger_type": {
+      "enum": ["count_rate", "image"],
```

Comment:
@jracusin please help with #127

Reply:
This is specifying if it's a rate or image trigger (just like Swift-BAT), not the count rate itself.  We have info in gcn.notices.core.statistics on rate and image SNR, duration, energy range.  I'd rather we add an enum to core.statistics with which type of trigger it is ["rate","image"] since it would be applicable to both BAT and SVOM.

Reply:
Yes you are right. It is specifying if its a rate or image trigger... I just thought it was clearer to specify "count_rate" instead of just "rate", but if it is a standard, I will modify.
However, even if the ECLAIRs trigger is a rate trigger, the SNR and other info are based on the image it produces on the sky, so we still have image_snr, etc.

Reply:
I just submitted PR #135 to add "trigger_type" to the core statistics schema.  @Vidushi-GitHub, please review.  Once that's merged in, it can be used from core here.  We will also have to do a minor version release.

Reply:
@timroland please use core.statistics.trigger_type